executor/logging.go

```diff
@@ -33,7 +33,7 @@ func bindLoggingPipe(name string, pipe io.Reader, output io.Writer, logPrefix bo
 			if logPrefix {
 				logger.Printf("%s: %s", name, scanner.Text())
 			} else {
-				logger.Printf(scanner.Text())
+				logger.Printf("%s", scanner.Text())
```

Comment:
I was expecting something like `logger.Print` so that there is no way to cause any formatting, but this works